### PR TITLE
Misc. updates

### PR DESCRIPTION
**Pull request by aiyengar2**

### CODE REVIEW

**Review by cbron — APPROVED** [review]
I think we already reviewed most of this, and proof will be in pudding.

File: templates/template/scripts/release-asset [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+# Copy assets into released-assets
+for asset in $@; do
+    asset=${asset#'assets/'}
+    mkdir -p $(dirname "released/assets/${asset}")
+    mv "to-be-released/${asset}" "released/assets/${asset}"
+done
+
+rm -rf to-be-released
+find assets -type d -empty -delete
+find charts -type d -empty -delete
```
Comment (aiyengar2) [inline templates/template/scripts/release-asset:63]:
```suggestion
find charts -type d -empty -delete

```

File: templates/template/scripts/release-assets [hunk 2]
```diff
@@ -5,16 +5,4 @@ cd $(dirname $0)
...
-    exit 0
-fi
-
-cp -R assets/ released/assets
-rm released/assets/README.md
-rm -R $(ls -1 -d charts/*/) $(ls -1 -d assets/*/)
-
-helm repo index --merge ./released/assets/index.yaml --url released/assets released/assets
-cp released/assets/index.yaml index.yaml
+find assets -type f -mindepth 2 | xargs ./scripts/release-asset
```
Comment (aiyengar2) [inline templates/template/scripts/release-assets:8]:
```suggestion
find assets -type f -mindepth 2 | xargs ./scripts/release-asset

```

File: init.sh [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    fi
+else
+    echo "Creating an empty configuration.yaml file."
+    echo -n "" > configuration.yaml
+    echo "You will need to run make docs manually after filling in the configuration.yaml"
+    echo "To add a template for Github Workflow based pull-requests, run the following script and update .github/workflows/pull-request.yaml manually"
+    echo "curl ${CBS_RAW_LINK}/templates/.github/workflows/pull-request.yaml --output .github/workflows/pull-request.yaml"
+    echo "To add a template for Github Workflow based automatic pushes, run the following script and update .github/workflows/push.yaml manually"
+    echo "curl ${CBS_RAW_LINK}/templates/.github/workflows/push.yaml --output .github/workflows/push.yaml"
+fi
```
Comment (aiyengar2) [inline init.sh:40]:
```suggestion
fi

```

**Review by aiyengar2 — COMMENTED** [review]
nit: add newlines

**Pull request merged**